services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationService.kt

```diff
@@ -329,12 +344,63 @@ open class ApplicationService(
             logger.info { "Application deleted, id=$id, alluid=$alluid userid=$userId" }
         }
 
-    private fun initAccessForApplication(
+    /** Creates access for all application contacts, excluding the current user. */
+    private fun initAccessOnInitialSend(
         application: ApplicationEntity,
         hanke: HankeEntity,
-        currentUserId: String
+        currentUserId: String,
+    ) {
+        if (featureFlags.isDisabled(Feature.USER_MANAGEMENT)) {
+            logger.info { "Feature ${Feature.USER_MANAGEMENT} disabled. No tokens created." }
+            return
+        }
```

Comment:
I think the feature was already checked in HankeKayttajaService and EmailSenderService.

Reply:
I wanted to prevent the call on hankeKayttajaService.getKayttajaByUserId as there is no hankekayttajas currently. I guess it is not dangerous to do that but it serves no purpose either.

Reply:
Checking the feature at several steps within the same call adds complexity. The added benefits should outweigh that. If we want to check the features here, do we still need the other checks?

Reply:
I guess the flag check in hankekayttajaservice.saveNewTokensFromApplication is now redundant. Removed it.